web/themes/custom/pineapple/components/_comments.twig

```diff
@@ -90,8 +90,6 @@
   ]) %}
 
   <footer class="comment-meta">
-    {{ user_picture }}
```

Comment:
Comment this out for now and add a comment saying it will need to be re added in a later task.